ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-pathfinder.cfg

```diff
@@ -87,9 +87,11 @@ router adaptive-virtual-topology
       path-selection load-balance LB-PROD-AVT-POLICY-MPLS-ONLY
    !
    profile PROD-AVT-POLICY-VIDEO
+      internet-exit policy ZSCALER-EXIT-POLICY-2
```

Comment:
this should not be there - need to fix it

Reply:
fixed